CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Conjur now Throw RoleNotFound error in k8s authenticator 
```

Comment:
`Conjur now raises a RoleNotFound when trying to authenticate a non-existing host in authn-k8s`